src/bci_build/package/__init__.py

```diff
@@ -146,35 +150,80 @@ def parse(val: str) -> OsVersion:
     def __str__(self) -> str:
         return str(self.value)
 
+    @property
+    def is_sles(self) -> bool:
+        try:
+            int(self.value)
```

Comment:
this might fail when we SLCC cohesive becomes SLES16. but we can fix that later.